src/Majora/Framework/Loader/Bridge/Form/Type/EntityCollectionType.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Majora\Framework\Loader\Bridge\Form\Type;
+
+use Majora\Framework\Loader\Bridge\LoaderBridgeInterface;
+use Majora\Framework\Model\EntityCollection;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Class EntityCollectionType.
+ */
+class EntityCollectionType extends AbstractType
+{
+    /**
+     * @var array
+     */
+    private $loaders = [];
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+
+        $choices = function (Options $options) {
+            if ($this->hasLoader($options['loader_alias']) === false) {
```

Comment:
This constraint could be defined as `OptionsResolver` field validation on `loader_alias` option definition; http://symfony.com/doc/current/components/options_resolver.html#value-validation